src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -488,11 +738,21 @@ def evaluate_step_by_step(self, step_size: float, inputs: dict = None):
 
         Examples
         --------
+        1st example
```

Comment:
Use "Example 1: Evaluate step by step" and *Example 2: Evaluate step by step with input field data"?
We want everything to be easily readable--with no guessing!